src/ape/utils/basemodel.py

```diff
@@ -121,16 +121,14 @@ class ManagerAccessMixin:
         "ConversionManager", injected_before_use()
     )
 
-    dependency_manager: ClassVar["DependencyManager"] = cast(
-        "DependencyManager", injected_before_use()
-    )
-
     network_manager: ClassVar["NetworkManager"] = cast("NetworkManager", injected_before_use())
 
     plugin_manager: ClassVar["PluginManager"] = cast("PluginManager", injected_before_use())
 
     project_manager: ClassVar["ProjectManager"] = cast("ProjectManager", injected_before_use())
```

Comment:
as mentioned previously, maybe makes more sense to rename to `.local_project` or just `.project`